README.md

```diff
@@ -39,28 +42,49 @@ https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-d
 
 
 
+ development
 ## Git Commands
 
+### https://youtu.be/kqtD5dpn9C8
+
 ### 1. Docs
 - [Article](https://dzone.com/articles/top-20-git-commands-with-examples)
 
 - [Cheat-sheet](https://phoenixnap.com/kb/wp-content/uploads/2021/11/git-commands-cheat-sheet-by-pnap-v2.pdf)
 
+
+### Resolving merge conflict 
+
+ HEAD
```

Comment:
Lets remove the HEAD line. Please refer to the merge conflict talk that we had if you are confused on how to do this.